### DISCUSSION TITLE
How to login user with private key only at web/client/login

### DISCUSSION THREAD

**Discussion by useronkel** [drakkan/sftpgo#1536] [Q&A]
Hi.
Is there a way to login a user with a private key only?
The settings for enabled_login_methods doesn't give a hint.
When I set up a user to only login with a public/private key, he/she has no options to provide a private key, only password is possible.

**Comment by gui-sousa** [comment]
Hello, I’m adding this because I have a similar question.

Is it possible to make the use of a private key mandatory during login? Even after configuring the keys, when accessing SFTP via clients like WinSCP or FileZilla, it allows both password + key login and also accepts login with just the password.

I would like to know if it’s possible to make key-based login mandatory.
